content/resources/tools.md

```diff
@@ -101,4 +103,4 @@ Mozilla SSL/TLS Configuration Generator for Servers (Apache, nginx, etc.) – ht
 
 SSL Labs: SSL and TLS Deployment Best Practices – https://github.com/ssllabs/research/wiki/SSL-and-TLS-Deployment-Best-Practices
 
-OWASP TLS Cheat Sheet – https://cheatsheetseries.owasp.org/cheatsheets/Transport_Layer_Protection_Cheat_Sheet.html
\ No newline at end of file
+OWASP TLS Cheat Sheet – https://cheatsheetseries.owasp.org/cheatsheets/Transport_Layer_Protection_Cheat_Sheet.html
```

Comment:
The destination page says:

> DEPRECATED: Transport Layer Protection Cheat Sheet[¶](https://cheatsheetseries.owasp.org/cheatsheets/Transport_Layer_Protection_Cheat_Sheet.html#deprecated-transport-layer-protection-cheat-sheet)
>
> The Transport Layer Protection Cheat Sheet has been deprecated.
>
> Please visit the [Transport Layer Security Cheat Sheet](https://cheatsheetseries.owasp.org/cheatsheets/Transport_Layer_Security_Cheat_Sheet.html) instead.

Should we point to that page instead and change the description?

Reply:
Dimitris, I am not clear what you are asking me, here. 
It was not me to add the OWASP TLS Cheet Sheet link that you mention....
I am not clear what you are talking about....

Reply:
> Dimitris, I am not clear what you are asking me, here. It was not me to add the OWASP TLS Cheet Sheet link that you mention.... I am not clear what you are talking about....

Your pull request contains a change in the OWASP TLS Cheet Sheet language.

Reply:
It was not me to add that link, but of course I agree with your proposed change to it ! :)
Just committed to my fork, please review again my PR here.